AnyAgent/InventoryManagement.cs

```diff
@@ -40,41 +42,65 @@ public string GetJobClass()
 
         public string GetStoreType()
         {
-            return _appConfig.AppSettings.Settings["StoreType"].Value;
+            try
+            {
+                Logger.MethodEntry(ILogExtensions.MethodLogLevel.Debug);
+                Logger.Trace($"StoreType {_appConfig.AppSettings.Settings["StoreType"].Value}");
+                Logger.MethodExit(ILogExtensions.MethodLogLevel.Debug);
+                return _appConfig.AppSettings.Settings["StoreType"].Value;
+            }
+            catch (Exception e)
+            {
+                Logger.Error($"Error Getting Store Type: {e.Message}");
```

Comment:
Maybe FlattenException here instead to ensure we get the stack trace and actual exception type?

Reply:
@RexWheeler - Done